pkg/server/server.go

```diff
@@ -335,9 +335,11 @@ func (s *Server) Check(ctx context.Context, req *openfgapb.CheckRequest) (*openf
 		TupleKey:             req.GetTupleKey(),
 		ContextualTuples:     req.ContextualTuples.GetTupleKeys(),
 		ResolutionMetadata: &graph.ResolutionMetadata{
-			Depth: s.resolveNodeLimit,
+			Depth:         s.resolveNodeLimit,
+			DatabaseReads: 0,
 		},
 	})
+	span.SetAttributes(attribute.Int64("db_reads", int64(resp.ResolutionMetadata.DatabaseReads)))
```

Comment:
I'd like us to establish some more conventions around naming metadata things for query metadata, because we should in general have these things for each of our query endpoints (e.g. ListObjects, StreamedListObjects, Check etc..).

Can we go with something more like `<method_prefix>_database_query_count`

`check_database_query_count`
`list_objects_database_query_count`
etc..

Reply:
Why do we need to add the prefix? The span name already is `Check`.

Reply:
We don't need the method, I guess I'm just asking for a better name beyond `db_reads`.

Can we rename to `database_query_count`